### ISSUE TITLE
auth_nonce is not working with auth_type:reauthenticate

### ISSUE THREAD

**Issue by greatSumini** [greatSumini/react-facebook-login#56]
related: https://github.com/keppelen/react-facebook-login/issues/329

**Comment by singhsterabhi** [comment]
Can I submit a PR for this issue?

**Comment by hamedazhar-giz** [comment]
@greatSumini Did you get any solution?

**Comment by greatSumini** [comment]
released in version 3.1.0 🚀 
in v3.1.0, you can solve it by using loginOptions.auth_nonce. like below!

```tsx
<FacebookLogin
  appId="{APP_ID}"
  loginOptions={{
    auth_type: 'reauthentication',
    // #1. you can pass string. or,
    auth_nonce: '{ANY_UNIQUE_STRING}',
    // #2. you can pass function which returns string
    auth_nonce: () => Date.now().toString()
  }}
/>
```

@hamedazhar-giz thanks for interest 😄
